src/ToDo.UI/App.xaml

```diff
@@ -13,19 +13,32 @@
 				<!-- Load WinUI resources -->
 				<XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
 
+				<!-- Load Uno.UI.Toolkit resources -->
+				<ToolkitResources xmlns="using:Uno.Toolkit.UI" />
+
 				<!-- Load Material Color Palette -->
-				<MaterialColors xmlns="using:Uno.Material"
-								OverrideSource="ms-appx:///Styles/ColorPaletteOverride.xaml" />
-				<MaterialFonts xmlns="using:Uno.Material"
-							   OverrideSource="ms-appx:///Styles/MaterialFontsOverride.xaml" />
+				<MaterialColorsV2 xmlns="using:Uno.Material" OverrideSource="ms-appx:///Styles/ColorPaletteOverride.xaml" />
 
-				<!-- Load the Material control resources -->
-				<MaterialResources xmlns="using:Uno.Material" WithImplicitStyles="True" />
-				<ToolkitResources xmlns="using:Uno.Toolkit.UI" />
+				<!-- Load Material FontFamilies -->
+				<MaterialFonts xmlns="using:Uno.Material" OverrideSource="ms-appx:///Styles/MaterialFontsOverride.xaml" />
+
+				<!-- Load Material Control Resources -->
+				<MaterialResourcesV2 xmlns="using:Uno.Material" WithImplicitStyles="True" />
 
-				<!-- Load Material Toolkit resources -->
-				<MaterialToolkitResources xmlns="using:Uno.Toolkit.UI.Material" />
+				<!-- Load Material Toolkit Resources -->
+				<MaterialToolkitResourcesV2 xmlns="using:Uno.Toolkit.UI.Material" WithImplicitStyles="True" />
 
+				<ResourceDictionary>
```

Comment:
@kazo0 / @NVLudwig  Only specific to this app for the OverlayColor (for the FlyoutLightDismissOverlayBackground) ? or can be added into themes ?
```
<ResourceDictionary.ThemeDictionaries>
						<ResourceDictionary x:Key="Light">
							<Color x:Key="OverlayColor">#51000000</Color>
						</ResourceDictionary>
						<ResourceDictionary x:Key="Dark">
							<Color x:Key="OverlayColor">#51FFFFFF</Color>
						</ResourceDictionary>
					</ResourceDictionary.ThemeDictionaries>
```

Reply:
This looks like an overlay used for Flyout and such...If so then yes you can keep it for all apps

Reply:
@kazo0 I think we can move the resources to Uno.Themes then

Reply:
I can follow this up with a PR for the overlay color